release/ray_release/job_manager/anyscale_job_manager.py

```diff
@@ -259,6 +270,54 @@ def run_and_wait(
         )
         return self._wait_job(timeout)
 
+    def get_last_ray_error_logs(self) -> Optional[str]:
+        globs = [
+            "dashboard.log",
+            "dashboard.err",
+            "dashboard_agent.log",
+            "runtime_env_agent.log",
+            "raylet.out",
+            "raylet.err",
+            "gcs_server.out",
+            "gcs_server.err",
+        ]
+        for glob in globs:
+            last_ray_logs = self._get_last_ray_error_logs(
+                self.cluster_manager.cluster_id,
+                glob,
+            )
+            if last_ray_logs:
+                return last_ray_logs
+        return None
+
+    def _get_last_ray_error_logs(self, cluster_id: int, glob: str) -> Optional[str]:
```

Comment:
could you use a different name (rather than just prepending a `_`)? like `get_last_ray_error_logs_for_cluster` or something?